lib/plugins/aws/deploy/lib/create-stack.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
                 this.provider.disableTransferAccelerationForCurrentDeploy();
               }
             }
+
+            const inactiveStateNames = new Set(['REVIEW_IN_PROGRESS']);
+            const isSingleStack = data.Stacks.length <= 1;
```

Comment:
Why do we need `isSingleStack` check? I don't think it's possible to have two stacks of same name deployed

Reply:
Thank you @medikoo! I have realized that I have been confusing Stacks and Events.

CloudFormation's `describeStacks` Request does not return Events information, but only Stack information. Therefore, the `isSingleStack` variable appears to be unnecessary."